README.md

```diff
@@ -1,6 +1,6 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
+This repo contains the projects I worked on for the first half of STEP.
 This work is based on the [Google Software Product Sprint](https://g.co/softwareproductsprint) program.
 
 To get started:
```

Comment:
If you don't need this part of the README anymore, consider deleting it.

Reply:
Thanks for reviewing all the code. I'll fix mistakes in a few days.